app/code/Magento/InventorySwatchesFrontendUi/Test/Mftf/Test/StorefrontDisplayCorrectMessageWhenChildProductSelectedForConfigProductTest.xml

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontDisplayCorrectMessageWhenChildProductSelectedForConfigProductTest">
```

Comment:
Instead of
```
<!--Assign product to shared catalog-->
<comment userInput="Assign product to shared catalog" stepKey="commentAssignProductSharedCatalog"/>
```
you can just use
`<!--Assign product to shared catalog-->`